api/proto/churner/churner.proto

```diff
@@ -33,6 +33,8 @@ message ChurnRequest {
 	//   - If any of the quorum fails to register, this entire request will fail.
 	// The IDs must be in range [0, 255].
 	repeated uint32 quorum_ids = 5;
+	// The Ethereum address (in hex like "0x123abcdef...") of the operator.
+	string operator_address = 6;
```

Comment:
This would be better to place at the top. It's just about having to change the field number, which is breaking.